griffinli.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@griffinli"
+project: "Smart Home Temperature Sensor"
+---
+
+## Summary
+
+A temperature sensor that keeps track of the temperature in a room over time and ideally integrates into HomeKit. It would track the temperature through the night with the goal of getting my dorm room (without AC and with a heater that's making it too warm but that I can't turn down) to the optimal temperature while sleeping. It'll have a display so that I can see the temperature by looking at it, in addition to its 24/7 monitoring. I'll try to also make it measure the rate at which temperature is changing and take into account the weather conditions (temperature and maybe wind as well) to help me figure out how much to open my window at night to get the optimal sleep temperature in my room.
+
+## Plan
+
+1. Order materials
+2. Get the instantaneous temperature sensing + display working
+3. (try to) Get the HomeKit integration working
+4. Write software that shows the 24/7 temperature
+5. Add weather data and temperature rate-of-change to the software
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 4  | https://www.amazon.com/Raspberry-Model-2019-Quad-Bluetooth/dp/B07TC2BK1X | $159.89  |
+| Raspberry Pi Case | https://www.amazon.com/Flirc-Raspberry-Pi-Case-Silver/dp/B07WG4DW52 | $15.95 |
+| Sandisk MicroSD Card | https://www.amazon.com/SanDisk-Extreme-microSDTM-Adapter-SDSQXCD-128G-GN6MA/dp/B09X7DNF6G | $21.00 |
```

Comment:
Fairly certain raspberry pi's dont support 128gb cards :( As a raspberry user myself, I highly reccomend [this one](https://www.amazon.com/SanDisk-Extreme-microSDHC-Adapter-SDSQXWG-032G-ANCMA/dp/B01K3X3CXU) - I use it myself and its amazing

Reply:
Looks like it's working on the pi 4 (https://forums.raspberrypi.com/viewtopic.php?t=268757) — I'll try it but if it doesn't work I can buy a smaller-sized one on my own